### PR TITLE
docs(QUICKSTART.md): created QUICKSTART.md

### PR DESCRIPTION
**Pull request by vxdguy**
Created QUICKSTART.md -- a starting point for someone new to this extension.

### CODE REVIEW

**Comment by ryanluker** [comment]
@vxdguy Excellent work and documentation on this!
I will link this to the ticket #333 , as it is related and gets us a good portion of the way to having some intro documentation.
I don't think we will close the above ticket yet as I still want to take a stab at the walkthroughs, but this can definitely go into release 2.10.0.

File: QUICKSTART/QUICKSTART.md [hunk 1]
```diff
@@ -0,0 +1,258 @@
...
+## Terminology
+
+*Coverage Gutters* depends uses two files.  It's easy to get the files mixed up and we need to make sure that everone is on the same page when we talk about these files.
+
+<details>
+<summary>Coverage File</summary>
+
+-----
+
+  * LCOV or XML file containing detailed coverage data
```
Comment (ryanluker) [inline QUICKSTART/QUICKSTART.md:15]:
```suggestion
 * LCOV or XML file containing detailed coverage data
```
minor, spacing improvement.

File: QUICKSTART/QUICKSTART.md [hunk 2]
```diff
@@ -0,0 +1,258 @@
...
+
+* *Coverage Gutters* supports LCOV, CLOVER, COBERTURA, and JACOCO coverage files
+
+* Generation of the coverage file is language specific and is beyond the scope of this extension.
```
Comment (ryanluker) [inline QUICKSTART/QUICKSTART.md:19]: Maybe mention after this that they could look at the examples for scenarios on how to generate the coverage files.
https://github.com/ryanluker/vscode-coverage-gutters/tree/master/example/ruby#ruby

File: QUICKSTART/QUICKSTART.md [hunk 3]
```diff
@@ -0,0 +1,258 @@
...
+
+</details>
+
+<details>
+<summary>Coverage Report</summary>
+
+-----
+
+  * HTML file(s) that provides a nice way to view coverage data.
```
Comment (ryanluker) [inline QUICKSTART/QUICKSTART.md:28]: These dropdown md sections are pretty cool.
Above I mentioned the extra space at the front of the first item, but I am thinking that is a part of the format now 😓.

File: QUICKSTART/QUICKSTART.md [hunk 4]
```diff
@@ -0,0 +1,258 @@
...
+
+* *Coverage Gutters* does **not** utilize this file.
+
+* Generation of the coverage report is language specific and beyond the scope of this extension.
+
+The extension assumes these two files are located somewhere in your workspace.  These are settings (covered later) to let you customize where **Coverage Gutters** look for both of these files.
```
Comment (ryanluker) [inline QUICKSTART/QUICKSTART.md:34]:
```suggestion
The extension assumes these two files are located somewhere in your workspace.  There are settings (covered later) to let you customize where **Coverage Gutters** will look for both of these files.
```

**Review by ryanluker — COMMENTED** [review]
Just some minor tweaks but looking really great so far!

File: QUICKSTART/QUICKSTART.md [hunk 5]
```diff
@@ -0,0 +1,258 @@
+
+# Quickstart for VSCODE-CODE-COVERAGE
```
Comment (ryanluker) [inline QUICKSTART/QUICKSTART.md:2]: We should link this document from the readme that is exposed on the vscode marketplace. (https://github.com/ryanluker/vscode-coverage-gutters/blob/master/README.md)

File: QUICKSTART/QUICKSTART.md (hunk 1 above)
Comment (ryanluker) [reply QUICKSTART/QUICKSTART.md:15]: nvm not needed

**Comment by EagleoutIce** [comment]
Is something missing here? Or why was this not merged as far as I can see?

**Comment by ryanluker** [comment]
> Is something missing here? Or why was this not merged as far as I can see?

Hmm after checking nothing was springing to mind, so I will go ahead and merge this!
Thanks for the poke!

**Review by ryanluker — APPROVED** [review]

**Comment by ryanluker** [comment]
@vxdguy Thanks for the contribution!

**Pull request merged**